sources/PTYTextView.m

```diff
@@ -4831,8 +4833,10 @@ - (void)doCommandBySelector:(SEL)aSelector
         // See comment in -keyDown:
         DLog(@"Rerouting doCommandBySelector from %@ to %@", self, gCurrentKeyEventTextView);
         [gCurrentKeyEventTextView doCommandBySelector:aSelector];
+        [self.delegate keyDown:_keydownEvent];
```

Comment:
To preserve behaviour.

For example, when press Enter key, current behaviour is following:

 1. A key event is passed to `interpretKeyEvents:`.
 2. `doCommandBySelector:` is called.
 3. Because `_inputMethodIsInserting` is `NO`,  `[delegate keyDown:_keydownEvent]`(at 1445 line) is called.

But at our pull request:

 1.  1. A key event is passed to `handleEvent:`.
 2. `doCommandBySelector:` is called. (same as current behaviour)
 3. `_inputMethodIsInserting` becomes `YES`, because input method handles it by calling `doCommandBySelector:`.
 4. `[delegate keyDown:_keydownEvent]`(at 1445 line) is not called.

To prevent this, we add this line.